### PR TITLE
frontends.tensorflow.nn: add supported for weighted ce with logits

### LINKED ISSUE #5107: weighted_cross_entropy_with_logits

### PR DESCRIPTION
**Pull request by hmahmood24**
Close #5107

### CODE REVIEW

**Comment by hmahmood24** [comment]
Hi @RashulChutani,

A soft reminder for you to review this PR and let me know if there are any issues with it so I can move towards opening other issue(s) and start working on them. I have gone through the logs of the lint and nightly tests and both of them pass on the function and unit test specific to this PR. Hoping to hear from you. 😊 Best.

**Comment by AnnaTz** [comment]
> Hi @RashulChutani,
> 
> A soft reminder for you to review this PR and let me know if there are any issues with it so I can move towards opening other issue(s) and start working on them. I have gone through the logs of the lint and nightly tests and both of them pass on the function and unit test specific to this PR. Hoping to hear from you. 😊 Best.

Hi @hmahmood24, stepping in for Rashul I think all's good with your PR 🙂. Ready to be merged!
